modules/basic-skills/src/backend/slot.ts

```diff
@@ -129,11 +129,20 @@ const createNodes = data => {
           name: 'builtin/setVariable {"type":"temp","name":"alreadyExtracted","value":"true"}'
         }
       ],
-      onReceive: undefined,
+      onReceive: [
+        data.validationAction && {
+          type: sdk.NodeActionType.RunAction,
+          name: `${data.validationAction} {}`
+        }
+      ],
```

Comment:
I'd suggest moving that part in a variable instead, like "onReceiveValidation", so you can use it here and in the previous place (line 32). No duplication then